### ISSUE TITLE
Mad?

### ISSUE THREAD

**Issue by Usedcondom9** [ShadowMario/FNF-PsychEngine#11138]
### What is your question?

![E9D593C9-5B57-4E89-B320-193BC4689C58](https://user-images.githubusercontent.com/117535391/200133153-67b37485-5418-4510-b543-f15b1e6f1bd7.png)

**reasoning:**

i hope you fucking are lmfao, you continued being toxic to half the people here, hayden hating children 
(despite him being 15 you hypocrite) and using useless insults you pulled out of peoples asses,
 let’s talk about the discord server then? Shall we not? 

“oooo if you say “why” your now a pibby modder go back to fnm you untalented piece of shit”

![17B3EB60-FCBD-4244-9E29-62F5F4F499F1](https://user-images.githubusercontent.com/117535391/200133263-e02e6074-b3d3-44cf-bb8a-8a6beb4ef5df.jpeg)

yeah… person on their computer who’s done that before, go ahead! be toxic to people for no reason!
that’s not how it works you idiot. you have to say “why” once in your life you massive shit smelling hypocrite.
ban me again I dare you fuckward, I bet you won’t even go far considering I’ve got this “copypasta” as a .txt

my initial plan was to get you to close this repo, but considering your so gone in your deep tf2 incel mindset,
You wont. Therefore, I took things into my own hands.

**list of alt accounts:**

NimudevfurryUwU
NimuFurrySoftUwU
TheKillerA1
GrandmasKisses
NotNimu
ifartedinmyteacgersdesk
Whatever you call that cat pfp thingy
This

and most importantly, my main

**jimmyjamescool**

Tailsic, we were never friends, Gabriel, fuck you clone, Hayden, you burn in hell you piece of shit, Average, kys.

fuck you all from what **they** did to me you worthless pieces of shit.

**teased convos (not using leaked you dumbass that’s not how it works**

![189D3044-0651-4A5C-A4F6-F088E82024E8](https://user-images.githubusercontent.com/117535391/200133800-9955339e-ed7b-4042-a1b8-2a1921c9247a.jpeg)
![76F3C407-07BB-43A2-AFB9-A6F039C2DB6C](https://user-images.githubusercontent.com/117535391/200133811-596be5c4-1798-41ef-aaf7-e706708d2e20.jpeg)
![4438464C-F930-4147-A285-F8687FC7503A](https://user-images.githubusercontent.com/117535391/200133813-cc3d5032-5804-400a-a632-c80857f3c601.jpeg)

Description: got too lazy to make the rest of this shit censored, so I just cut it out.
This was the first argument that managed to make me who I am now, and by being a dumbass, I managed to troll this repo.

Don’t you dare fucking ratio me as I bet your gonna get 2 uppers
We don’t need anymore examples.

**deaths:** 

my dog died in 2021,
My mom died in 2022,
My aunt died in 2019,

Description: not much to say here other than the fact it kinda impacted the way I am.

**In conclusion:** fuck you all

![955E763F-41F5-4394-90DC-F0DDD3274DE7](https://user-images.githubusercontent.com/117535391/200134059-5d07d373-de84-4797-821d-a190d937e9ff.gif)

**Comment by MemeHoovy** [comment]
I find it funny how their github got deleted as soon as they posted this

**Comment by MemeHoovy** [comment]
like Jimmy, find something to do

**Comment by MemeHoovy** [comment]
okay looking at this again, it doesn't look like they were throwing insults towards me at all, just other people 🤷🏻‍♂️

**Comment by CaptainBaldi** [comment]
honestly i cant find any other emotion for this post but laughter
stay mad lol

**Comment by AverageDaveLover65** [comment]
hello jimmy
honestly im not affected by your shitty words
also i made a mockinjg out of you because i thoguht it was funyny! !!  !
also you can kill yourself, i dont believe a single shit (except for the alts, that was obvious), stop trying to pull a cycba (guilt tripping)

also dont think youre the victim here, theres some evidence and shit that follow up to you possibly being a groomer or any shit, hell, even being the cunt you are.
dont think youre a chad, youre an virgin. please honestly get off the internet and stop trying to get this repo to be closed, it ain't honestly funny so please go fuck yourself :chad:

also i dont CARE about words on my screen, I DONT FUCKING CARE ABOUT HOW YOU THINK OF ME, BUT EVERYWHERE YOU GO, YOU WILL REMEMBER EVERYTHING YOUVE DONE, AND YOU WILL PAY FOR IT. YOU UNDERSTAND YOU FUCKING BITCH?

thats right, stay dead silent you fucker.

**Comment by Daikicalla** [comment]
Y'all are dumb as hell

**Comment by AverageDaveLover65** [comment]
> > Y'all are dumb as hell
> 
> fr fr, average is always like this idk why.

he said "yall" not me speicficically
